microsoft/testsuites/device_passthrough/functional_tests.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path
+from typing import Any, Dict, cast
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.sut_orchestrator import CLOUD_HYPERVISOR
+from lisa.sut_orchestrator.libvirt.ch_platform import CloudHypervisorPlatform
+from lisa.sut_orchestrator.libvirt.context import get_node_context as get_ch_context
+from lisa.sut_orchestrator.libvirt.schema import BaseLibvirtNodeSchema
+from lisa.testsuite import TestResult, simple_requirement
+from lisa.tools import Lspci
+from lisa.util import LisaException, SkippedException
+
+
+@TestSuiteMetadata(
+    area="device_passthrough",
+    category="functional",
+    description="""
+    This test suite is for testing device passthrough functional tests.
+    """,
+)
+class DevicePassthroughFunctionalTests(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            Check if passthrough device is visible to guest.
+            This testcase support only on CLOUD_HYPERVISOR
+            platform of LISA.
+        """,
+        priority=4,
+        requirement=simple_requirement(
+            supported_platform_type=[CLOUD_HYPERVISOR],
```

Comment:
Yes @squirrelsc , Idea is to extend the same test for hyper-v too. As of now, this test is solely for CH platform. 

We need some changes to make the test work on hyper-v platform and I cannot test the change because of some infra issue on hyper-v. So, that is scoped for future one.